src/lib/common/Format.h

```diff
@@ -31,6 +31,15 @@
 
```

Comment:
Despite the description of the PR as "notifyFormat now fully working" I understand that it also finish the implementation of the format unhardwiring for CPrs forwarding. Is that correct?

(Probably I will know the answer after reviewing all the PR, but I prefer to include the comment, just to not forget the doubt :)